examples/helloworld/helloworld_test.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+set +e
+
+clean () {
+    if [ ! -z $PID1 ]; then
+        kill $PID1
```

Comment:
It seems that `pkill -P $$` doesn't kill the process correctly, the test fails on a second run because the server can't start again. I decided to use `fuser -k 50051/tcp` to directly end the process running on the port number.

Reply:
It looks like the `pkill -P $$` only kills the `go run` process, not the child it forked to actually run the server executable.

`pkill -s 0` seems more like the way to go.  That will kill both.

The reason I would prefer not to specify the port and use `fuser -k` is to make it more compatible with other examples that might be using multiple servers or may have different ports bound.

Reply:
Done.